fr/_posts/2022-07-25-synchronisation-des-communiques.md

```diff
@@ -0,0 +1,135 @@
+
+[Alertes contextuelles – Modèle de conception de Canada.ca](https://conception.canada.ca/configurations-conception-communes/alertes-contextuelles.html)
+
+**Placer les alertes dans le contexte, pas seulement en haut**
+
+Examinez attentivement le placement des alertes. Les défileurs enthousiastes peuvent manquer une alerte en haut d’une page. En la plaçant dans la sous-section pertinente d’une page, vous attirez l’attention sur un changement important au point où il est le plus logique pour le lecteur. Cela maximise l’incidence de l’alerte.
+
+**Améliorez la conception de votre contenu d’alerte**
+
+Le placement d’alertes n’est que la moitié du combat. Votre conception peut faire toute la différence dans son efficacité.
+
+- **Niveau de lecture :** Écrivez le texte d’alerte à un niveau de lecture de la 6e à la 8e année. Vous voulez que les gens comprennent facilement pourquoi ils doivent être attentifs.
+- **Longueur :** Gardez le texte d’alerte au minimum. Les longues alertes qui contiennent trop de renseignements sont distrayantes et agaçantes. Ceci est particulièrement vrai sur la version mobile, où ils poussent le contenu principal vers le bas de l’écran. 
+- **Date d’entrée en vigueur :** Indiquez la date exacte à laquelle les modifications entrent en vigueur.
+- **Temps du verbe :** Faites attention au temps de votre verbe. Si les changements ne sont pas en vigueur pendant une autre semaine, n’écrivez pas dans le présent. Mettez votre texte d’alerte au futur, et mettez-le à jour une fois la date d’entrée en vigueur passée. 
+
+<figure class="gc-complex-img" role="group">
+<img alt="Une longue description peut être trouvée après l'image."
+src="/images/Alerte-voyage-1.png">
+</figure>
+<details>
+<summary>À faire : date claire, temps de verbe exact, répété dans le contexte de la page</summary>
+<p>Le widget affiche la question&nbsp;: «&nbsp;Avez-vous trouvé ce que vous cherchiez?&nbsp;» et propose des options oui et non.</p>
+</details>
+
+
+
+<figure class="gc-complex-img" role="group">
+<img alt="Une longue description peut être trouvée après l'image."
+src="/images/Alerte-voyage-2.png" >
+</figure>
+<details>
+<summary>À faire : date claire, temps de verbe exact, répété dans le contexte de la page</summary>
+<p>Le widget affiche la question&nbsp;: «&nbsp;Avez-vous trouvé ce que vous cherchiez?&nbsp;» et propose des options oui et non.</p>
+</details>
+
+
+
+<figure class="gc-complex-img" role="group">
+<img alt="Une longue description peut être trouvée après l'image."
+src="/images/Alerte-voyage-3.png" >
+</figure>
+<details>
+<summary>À ne pas faire : La date critique était manquante et les verbes sont au présent, il a donc semblé que les règles étaient déjà en vigueur (publiées le 15 février, les règles n’ont pas changé avant le 28 février)</summary>
+<p>Le widget affiche la question&nbsp;: «&nbsp;Avez-vous trouvé ce que vous cherchiez?&nbsp;» et propose des options oui et non.</p>
+</details>

```

Comment:
We need to put the details/summary inside a figcaption tag inside the figure tag (see previous comment)